openhands/runtime/action_execution_server.py

```diff
@@ -420,6 +420,10 @@ def close(self):
         self.browser.close()
 
 
+def _allow_entry(entry: str) -> bool:
```

Comment:
I actually think this should go here: https://github.com/All-Hands-AI/OpenHands/blob/main/openhands/server/file_config.py#L8

If the agent runs `ls` it'll still see lost+found, but I think that's OK/desirable